querybook/webapp/lib/data-doc/datadoc-socketio.ts

```diff
@@ -66,6 +66,7 @@ export interface IDataDocSocketEvent {
         (
             queryExecution: IQueryExecution,
             dataCellId: number,
+            fromDataDocRun: boolean,
```

Comment:
is fromDataDocRun needed?

Reply:
actually not needed any more as we return originator as None. the sameOrigin will be false anyway.